release_dashboard/test/fakes/services/fake_conductor.dart

```diff
@@ -19,10 +21,12 @@ import '../fake_start_context.dart';
 /// release dashboard.
 class FakeConductor extends ConductorService {
   FakeConductor({
+    this.fakeCleanContextProvided,
```

Comment:
FakeCleanContext should be constant, so you can do `this.fakeCleanContext = FakeCleanContext();` and make the field non-nullable